esp-hal/README.md

```diff
@@ -7,15 +7,33 @@
 
 `no_std` HAL implementations for the peripherals which are common among Espressif devices. Implements a number of the traits defined by [embedded-hal](https://github.com/rust-embedded/embedded-hal).
 
-This crate should not be used directly; you should use one of the device-specific HAL crates instead:
-
-- [esp32-hal](../esp32-hal/README.md)
-- [esp32c2-hal](../esp32c2-hal/README.md)
-- [esp32c3-hal](../esp32c3-hal/README.md)
-- [esp32c6-hal](../esp32c6-hal/README.md)
-- [esp32h2-hal](../esp32h2-hal/README.md)
-- [esp32s2-hal](../esp32s2-hal/README.md)
-- [esp32s3-hal](../esp32s3-hal/README.md)
+[esp-hal] encompasses the hardware abstraction layer across multiple Espressif devices atop peripheral access crate. To select a concrete chip, correct chip feature has to be used. The current list of chip features is `esp32, esp32c2, esp32c3, esp32c6, esp32h2, esp32p4, esp32s2, esp32s3`.
```

Comment:
Not sure about this list feature, we have it https://github.com/esp-rs/esp-hal?tab=readme-ov-file#esp-hal-1 as well 🤔

Reply:
This goes in the code documentation too, so I think this is worth having here.

Reply:
IMHO there is no need to repeat it here

Reply:
ah yeah - the README is also for crates.io 👍